Makefile

```diff
@@ -489,10 +494,31 @@ verify-links:
 	fi;
 .PHONY: verify-links
 
-verify: verify-gofmt verify-codegen verify-crds verify-helm-schemas verify-helm-charts verify-deploy verify-examples verify-govet verify-helm-lint verify-links
+# $1 - source file
+# $2 - client directory
+# $3 - client name
+define generate-swagger-client
+	GOPATH=$(GOPATH) $(GO) run "$(SWAGGER)" generate client --name "$(3)" --spec "$(1)" --target "$(2)"
```

Comment:
should be simple quotes, you don't want the substitutions to evaluate bash vars and make vars are evaluated before
(applies globally)